src/main/java/org/pesmypetcare/webservice/entity/petmanager/PetEntity.java

```diff
@@ -232,13 +246,14 @@ public static void checkIllnesses(String key, Map<String, Object> body) {
      */
     public static void checkMedications(String key, Map<String, Object> body) {
         checkDatePlusNameFormat(key);
-        if (body.size() != 3 || !body.containsKey("quantity") || !body.containsKey("duration")
-            || !body.containsKey("periodicity")) {
-            throw new IllegalArgumentException("Request body does not have a correct format");
+        String periodicity = "periodicity";
+        if (body.size() != 3 || !body.containsKey(QUANTITY) || !body.containsKey(DURATION)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['3' is a magic number.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5492924)